concordium-consensus/tests/globalstate/GlobalStateTests/Instances.hs

```diff
@@ -15,29 +15,41 @@ import Data.FileEmbed
 import qualified Concordium.Crypto.SHA256 as H
 import Concordium.Types
 import qualified Concordium.Wasm as Wasm
-import qualified Concordium.Scheduler.WasmIntegration as WasmIntegration
+import qualified Concordium.Scheduler.WasmIntegration as WasmV0
+import qualified Concordium.Scheduler.WasmIntegration.V1 as WasmV1
 import qualified Concordium.GlobalState.Wasm as GSWasm
 import Concordium.Types.HashableTo
 import Concordium.GlobalState.Basic.BlockState.InstanceTable
 import Concordium.GlobalState.Basic.BlockState.Instances
+import Concordium.GlobalState.Instance
 
 import qualified Data.FixedByteString as FBS
 import qualified Data.ByteString as BS
 
 import Test.QuickCheck
 import Test.Hspec
 
-contractSources :: [(FilePath, BS.ByteString)]
-contractSources = $(makeRelativeToProject "testdata/contracts/" >>= embedDir)
+contractSourcesV0 :: [(FilePath, BS.ByteString)]
+contractSourcesV0 = $(makeRelativeToProject "testdata/contracts/" >>= embedDir)
```

Comment:
I don't want to move files as part of a larger PR since it makes merging more error prone.

It should be done separately in the future, I agree with that.